Client/Frontend/Browser/Browser.swift

```diff
@@ -26,6 +26,8 @@ class Browser: NSObject {
     var browserDelegate: BrowserDelegate? = nil
     var bars = [SnackBar]()
     var favicons = [Favicon]()
+    var urls: [NSURL]? = nil
+    var currentPage = 0 //arbitrary
```

Comment:
Let's make a `SessionData` class or struct that holds these rather than including them as separate variables.